src/Controller/User/UserController.php

```diff
@@ -154,7 +154,7 @@ public function showAction(User $targetUser)
     public function editAction(Request $request, User $targetUser)
     {
         $deleteForm = $this->createDeleteForm($targetUser);
-        $editForm = $this->createForm(UserType::class, $targetUser);
+        $editForm = $this->createForm(UserType::class, $targetUser, ['validation_groups' => ['registration']]);
```

Comment:
This should **not** have the validation group `registration` added as adding it means that a new password for the user would be **required** when any *other change* to the user is made (such as changing their role).  One should be able to change the user's role without changing their password.